packages/rest/src/__tests__/unit/router/routing-table.unit.ts

```diff
@@ -214,6 +214,44 @@ function runTestsWithRouter(router: RestRouter) {
     expect(route.pathParams).to.containEql({userId: '1', format: 'json'});
   });
 
+  it('finds "GET /orders/ and GET /orders/{id}" endpoints', () => {
```

Comment:
```suggestion
  it('finds "GET /orders and GET /orders/{id}" endpoints', () => {
```

The test names should match the code.